backend/urls.py

```diff
@@ -34,74 +35,77 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
 
-    path('api/bodies', BodyViewSet.as_view({'get':'list', 'post':'create'})),
+    path('api/bodies', BodyViewSet.as_view({'get': 'list', 'post': 'create'})),
     path('api/bodies/<pk>', BodyViewSet.as_view(
```

Comment:
No unnecessary whitespace changes please; make a separate PR to master for this if necessary.